### ISSUE TITLE
Acronym Explorer

### ISSUE THREAD

**Issue by squarefrog** [RocketFanOrg/RocketFanApp#86]
There are a tonne of acronyms used throughout the API. It might be nice to show what these acronyms mean. Attached are some JSON files with various acronyms in.

There are a couple of ways we could approach this.

1. Have a new filterable list view somewhere (possibly About/Settings?) which allows you to search for the acronym you're looking for

1. Actively look for acronyms in text fields, and allow the text to be tappable. When tapped show a popover with the acronym and description.

[acronyms.zip](https://github.com/RocketFanOrg/RocketFanApp/files/3595180/acronyms.zip)

**Comment by RJ-Clegg** [comment]
I think the second approach is better for discovery. If we highlight the acronyms, to show that they're tappable, it should be used more often than if we bury it in settings?

**Comment by squarefrog** [comment]
Yes this is true, but it would be much harder to achieve. It would involve all of the text fields being attributed text, scanning for any acronym and creating links. 

Not sure how possible this is with `SwiftUI` at the moment, but I do agree the latter is a better experience.
